components/admin/lessons/AdminLessonNav.tsx

```diff
@@ -1,18 +1,22 @@
-import React from 'react'
```

Comment:
I think `AdminLessonSideNav` is a better name for this component because its a bit more clear. Whenever I see `AdminLessonNav` I think about the horizontal nav.
<img width="483" alt="Screen Shot 2022-06-07 at 2 31 39 PM" src="https://user-images.githubusercontent.com/686933/172486386-7a37e400-2742-4897-bdae-67b612837d04.png">